app/infrastructure/parsers/lms_parser.py

```
@@ -0,0 +1,301 @@
+        data = {
+            "type": activity_type,
+            "title": self._get_title(element=element),
+            "url": url,
+            "available": available,
+        }
+
+        if url:
+            try:
+                data["id"] = UrlUtils.extract_value_from_url(url=url, param_name="id")
+            except ValueError:
+                pass
```

Ruff (0.12.2)

Use `contextlib.suppress(ValueError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(ValueError)`

(SIM105)